en-US/Grammar.xml

```diff
@@ -471,10 +471,15 @@
 				</thead>
 				 <tbody>
 					<row>
-						<entry> The individual member of the social community often receives information via visual, symbolic channels. </entry>
-						 <entry> People read. (Translation by Richard Feynman.) </entry>
+						<entry> The <command>fsck</command> utility performs a file system verification on the XFS file system residing on the <filename>/dev/vdb1</filename> partition. </entry>
+						 <entry> The <command>fsck</command> utility verifies the XFS file system on the <filename>/dev/vdb1</filename> partition. </entry>
```

Comment:
The example is much better (I've been meaning to fix this for ages) but it's also not quite accurate. If you run `fsck` with no args it checks all file systems on all devices. If you want to check a specific file system then you have to pass it as an arg. E.g., `fsck /dev/hd1`

Reply:
Updated the command to include arguments.

Reply:
@psweany or @bonnevil comment please?